web/src/routes/(user)/albums/+page.svelte

```diff
@@ -70,7 +70,9 @@
 			</div>
 
 			<!-- Album Card -->
-			<div class="flex flex-wrap gap-8">
+			<div
+				class="grid sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-3 xl:grid-cols-4 2xl:grid-cols-5 3xl:grid-cols-6 4xl:grid-cols-7 5xl:grid-cols-8 gap-8"
```

Comment:
Maybe better to use something like `grid gap-8 grid-cols-[repeat(auto-fill,minmax(15rem,1fr))]` instead of defining the amount of columns for every breakpoint and even having to add breakpoints